From df5a746bba9290e7eb6ca17a88ee28feea312b6e Mon Sep 17 00:00:00 2001
From: David S. Wang <dsw@cloudera.com>
Date: Wed, 27 Mar 2013 08:26:18 -0700
Subject: [PATCH 03/96] HBASE-5740 Compaction interruption may be due to balancing

Reason: Bug
Author: Michael Stack, Jimmy Xiang
Ref: CDH-4963
(cherry picked from commit dc138376295affe507cde71178a5f20c9066691d)
---
 .../hadoop/hbase/regionserver/Compactor.java       |    2 +-
 .../apache/hadoop/hbase/regionserver/HRegion.java  |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/main/java/org/apache/hadoop/hbase/regionserver/Compactor.java b/src/main/java/org/apache/hadoop/hbase/regionserver/Compactor.java
index 900a137..0cb0345 100644
--- a/src/main/java/org/apache/hadoop/hbase/regionserver/Compactor.java
+++ b/src/main/java/org/apache/hadoop/hbase/regionserver/Compactor.java
@@ -228,7 +228,7 @@ class Compactor extends Configured {
     writer.close();
     store.getFileSystem().delete(writer.getPath(), false);
     throw new InterruptedIOException( "Aborting compaction of store " + store +
-      " in region " + store.getHRegion() + " because user requested stop.");
+      " in region " + store.getHRegion() + " because it was interrupted.");
   }
 
   CompactionProgress getProgress() {
diff --git a/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java b/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java
index 6eb5f5e..593b4ba 100644
--- a/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java
+++ b/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java
@@ -1326,7 +1326,7 @@ public class HRegion implements HeapSize { // , Writable{
           status.setStatus("Compacting store " + cr.getStore());
           cr.getStore().compact(cr);
         } catch (InterruptedIOException iioe) {
-          String msg = "compaction interrupted by user";
+          String msg = "compaction interrupted";
           LOG.info(msg, iioe);
           status.abort(msg);
           return false;
-- 
1.7.0.4

